app/networkapi/people/models.py

```diff
@@ -97,6 +97,12 @@ class Person(SortableMixin):
         null=True,
         blank=True,
     )
+    interview_url = models.URLField(
+        max_length=500,
```

Comment:
Also, Stackoverflow led me to a a conversation that's somewhat relevant. https://code.djangoproject.com/ticket/19515. So I think I'm going to keep this for now, as I know 500 chars works; If someone needs to post a longer url, they can use a shortener.